tests/macsec/test_dataplane.py

```diff
@@ -120,18 +120,59 @@ def test_neighbor_to_neighbor(self, duthost, ctrl_links, upstream_links, wait_mk
                     requester["peer_ipv4_addr"]), module_ignore_errors=True)
 
     def test_counters(self, duthost, ctrl_links, upstream_links, rekey_period, wait_mka_establish):
-        if rekey_period:
-            pytest.skip("Counter increase is not guaranteed in case rekey is happening")
-        EGRESS_SA_COUNTERS = (
+
+        def get_counters(duthost, up_ports):
+            egress_counters = Counter()
+            ingress_counters = Counter()
+            for up_port in up_ports:
+
+                asic = duthost.get_port_asic_instance(up_port)
```

Comment:
I think we do not have get te asic instance here the. `show macsec` works on the linecard level